api/openapi-spec/v1.0.yaml

```diff
@@ -91,6 +91,56 @@ paths:
         default:
           $ref: '#/components/responses/error'
       x-ms-docs-operation-type: operation
+  /v1.0/me/photo:
+    get:
+      tags:
+        - me.photo
+      summary: Get the current user's profile photo
+      operationId: GetOwnUserPhoto
+      responses:
+        '200':
+          description: Retrieved entity
+          content:
+            image/png:
+              schema:
+                type: string
+                format: binary
+        default:
+          $ref: '#/components/responses/error'
+    put:
+      tags:
+        - me.photo
+      summary: Update the current user's profile photo
+      operationId: UpdateOwnUserPhotoPut
+      requestBody:
+        description: New user photo
+        content:
+          image/png:
+            schema:
+              type: string
+              format: binary
+      responses:
+        '204':
```

Comment:
MS Graph seems to return 200 here (https://learn.microsoft.com/en-us/graph/api/profilephoto-update?view=graph-rest-1.0&tabs=http#example-1-update-the-photo-of-a-user). Though that's probably not really correct when the response as no body?

Reply:
204 makes more sense to me here